### DISCUSSION TITLE
onRowChange Event? Better onCellChange?

### DISCUSSION THREAD

**Discussion by ghost** [Comcast/react-data-grid#2314] [Q&A] [answered]
am currently testing react-data-grid. Unfortunately, I can't find an event for RowChange within.

Background: I have a cell and after a change or onRowChange in the editor I only want to get back the row in the main control and not all rows as in the onRowsChange event ...

thanks for help.

**Comment by nstepien** [comment, marked as answer]
We've added a second parameter to `onRowsChange`, it's an object containing the `indexes` of the changed rows, and the `column` that triggered the change.
